### PR TITLE
Np 44791 Move Contributor verificationStatus

### PR DESCRIPTION
**Pull request by monaullah**
Moved contributorVerificationStatus from Contributor to Contributor.Identity

### CODE REVIEW

**Review by anetteOlli — APPROVED** [review]

File: documentation/schema.yaml [hunk 1]
```diff
@@ -780,9 +780,6 @@ referencedSchemas:
         extent:
           type: string
           example: null
-        description:
```
Comment (hackon) [inline documentation/schema.yaml:783]: It's gone description is?

File: nva-datamodel-java/src/main/java/no/unit/nva/model/Identity.java [hunk 2]
```diff
@@ -68,21 +85,19 @@ public boolean equals(Object o) {
...
     }
 
     public static final class Builder {
+
         private URI id;
         private String name;
         private NameType nameType;
         private String orcId;
 
+        private ContributorVerificationStatus verificationStatus;
```
Comment (hackon) [inline nva-datamodel-java/src/main/java/no/unit/nva/model/Identity.java:99]: Space above?

**Review by hackon — DISMISSED** [review]

File: documentation/schema.yaml (hunk 1 above)
Comment (monaullah) [reply documentation/schema.yaml:783]: Yes, Rurik removed in previous commit to main https://github.com/BIBSYSDEV/nva-datamodel-java/commit/7f13a7b513b6cf996f277f7ad59bac2e90ec1308

File: nva-datamodel-java/src/main/java/no/unit/nva/model/Identity.java (hunk 2 above)
Comment (monaullah) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/Identity.java:98]: Removed in new commit

**Review by brinxmat — APPROVED** [review]

**Pull request merged**